cmd/minishift/cmd/util/util.go

```diff
@@ -170,7 +170,7 @@ func ValidateGenericDriverFlags(remoteIPAddress, remoteSSHUser, sshKeyToConnectR
 // OcClusterDown stop Openshift cluster using oc binary inside the remote machine
 func OcClusterDown(hostVm *host.Host) error {
 	sshCommander := provision.GenericSSHCommander{Driver: hostVm.Driver}
-	cmd := fmt.Sprintf("%s/oc cluster down", minishiftConstants.OcPathInsideVM)
+	cmd := fmt.Sprintf("sudo %s/oc cluster down", minishiftConstants.OcPathInsideVM)
```

Comment:
Is this necessary?

Reply:
@gbraad yes, since `oc` command for `cluster up` and `cluster down` require access to docker daemon which `sudo` have access.

Reply:
this one needs, but some other `oc` commands, like `oc adm` needs?

Reply:
@gbraad commands which is going to use `/var/lib/minishift/bin/oc` need to use `sudo` because the `sudo oc cluster up` create the kubeconfig entry to `/root/.kube` instead `$HOME/.kube`. `oc` command which is used outside the cluster like from the host doesn't need `sudo` since that already logged in.